docs/source/optimizations/hilbert/hilbert_compile_css.ipynb

```diff
@@ -0,0 +1,342 @@
+   "id": "ae0dfcb6",
+   "metadata": {},
+   "source": [
+    "To interface Superstaq via Cirq, we must first instantiate a service provider in ```cirq-superstaq``` with ```Service()```. We then supply a Superstaq API key (which you can get from https://superstaq.super.tech.) by either providing the API key as an argument of Service, i.e. : ```css.Service(api_key=\"token\")``` , or by setting it as an environment variable. (see more details [here](https://docs-superstaq.readthedocs.io/en/stable/get_started/basics/basics_css.html#Set-up-access-to-Superstaq's-API)) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "4464f34f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "service = css.Service()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "415812a3",
+   "metadata": {},
+   "source": [
+    "## Create a Circuit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5d5ca21",
+   "metadata": {},
+   "source": [
+    "First, we will create an example Cirq circuit that we can then compile and submit onto Hilbert"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "9521d540",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0: ───H───@───M───\n",
+      "          │\n",
+      "1: ───────X───────\n"
+     ]
+    }
+   ],
+   "source": [
+    "qubits = cirq.LineQubit.range(16)\n",
+    "circuit1 = cirq.Circuit(cirq.H(qubits[0]), cirq.CNOT(qubits[0], qubits[1]), cirq.measure(qubits[0]))\n",
+    "print(circuit1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7812e22",
+   "metadata": {},
+   "source": [
+    "## Single Circuit Compilation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "b9149284",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "compiler_output = service.cq_compile(circuit1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "92f3d368",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(0, 0): ───RGate(0.5π, -0.5π)───@───RGate(-0.25π, -0.5π)───────RGate(0.25π, -0.5π)───────M('q(0)')───\n",
+      "           │                    │   │                          │\n",
+      "(0, 1): ───#2───────────────────@───#2─────────────────────Z───#2────────────────────Z───────────────\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(compiler_output.circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65c6a0f6",
+   "metadata": {},
+   "source": [
+    "If you would like to compile (or submit) on a different number of qubits, this can be done via the ```grid_shape``` option. This simply sets the shape of the rectangular qubit grid. However, specifying a grid that is incompatable with Hilbert's current capabilities will result in an error. "
```

Comment:
After some discussion with @vtomole: let's have the docs exclude `grid_shape` for now, but add it back once the [`grid_qubits` issue](https://github.com/Infleqtion/client-superstaq/issues/582#issuecomment-1614871528) is fixed